app/templates/public/schedule.hbs

```diff
@@ -1 +1 @@
-{{outlet}}
+<div class="ui disabled header">*Schedule for the event is currently not available</div>
```

Comment:
Remove the '*' used before Schedule. 
Use 'unavailable' in place of not available.